.github/ISSUE_TEMPLATE.md

```diff
@@ -17,6 +17,12 @@
 
 #### Screenshot / Video
 
+
+#### Context / Source
+<!-- Optional: Share with us your perspective and source, using suggested flow and source keywords here: https://github.com/Automattic/wp-calypso/blob/master/.github/CONTRIBUTING.md#reporting-bugs-asking-questions-sending-suggestions. -->
```

Comment:
Yes, good point — I'll update so the tags are here in the comment so it's easier.